leapmicro.html

```diff
@@ -0,0 +1,6 @@
+---
+title: openSUSE Leap Micro
+permalink: /leapmicro/
+image: /assets/images/logos/leapmicro_header.png
+---
+{% include leapmicro.html type="current" %}
```

Comment:
This won't work without 51.yml

Reply:
With 51.yml in place, you should have more useful warnings if you reuse distribution.html instead of distribution_leapmicro.html

Reply:
Ah to the leapmicro.html we've dropped it, and leapmicro_testing.html references type="testing". Therefore also resolved.